README.md

```diff
@@ -95,22 +95,30 @@ Contains all the inodes of the partition. The maximum number of inodes is equal
                                   127 | 0         |
                                       +-----------+
   ```
-  - for a file: the list of blocks containing the actual data of this file. Since block IDs are stored as 32-bit values, at most 1024 links fit in a single block, limiting the size of a file to 4 MiB.
+  - for a file: the list of blocks containing the actual data of this file. Since block IDs are stored as `sizeof(struct simplefs_extent)` bytes values, at most 341 links fit in a single block, limiting the size of a file to around 10.65 MiB (10912 KiB).
```

Comment:
Should `struct simplefs_extent` be aligned?

Reply:
Add overview about extent support as well.

Reply:
Do you mean to align to factor of block size? 
I think it doesn't matter.

Reply:
Use [IOzone](http://www.iozone.org/) for microbenchmarking.

Reply:
It does not seem to have a impact on performance
The capacity of my RAM is 4G
- naive
```
Command line used: iozone –Ra –g 8G –i 0 –i 1
	Output is in kBytes/sec
	Time Resolution = 0.000001 seconds.
	Processor cache size set to 1024 kBytes.
	Processor cache line size set to 32 bytes.
	File stride size set to 17 * record size.
                                                              random    random     bkwd    record    stride                                    
              kB  reclen    write  rewrite    read    reread    read     write     read   rewrite      read   fwrite frewrite    fread  freread
             512       4   416598  1071167  2307123  1683237  1413929   724325  1590957    920558   1091308   653961  1279935  1089094   864594
```
- pad to 16 byte
```cpp
struct simplefs_file_ei_block {
    struct simplefs_extent extents[SIMPLEFS_MAX_EXTENTS];
    char padding[4];
};
```
```
Command line used: iozone –Ra –g 8G –i 0 –i 1
	Output is in kBytes/sec
	Time Resolution = 0.000001 seconds.
	Processor cache size set to 1024 kBytes.
	Processor cache line size set to 32 bytes.
	File stride size set to 17 * record size.
                                                              random    random     bkwd    record    stride                                    
              kB  reclen    write  rewrite    read    reread    read     write     read   rewrite      read   fwrite frewrite    fread  freread
             512       4   254085   248040  1096322  1309596   740050   377559   949875    513039   1020276   663252   181546  
```